python/tidy/servo_tidy/tidy.py

```diff
@@ -827,6 +830,17 @@ def check_config_file(config_file, print_text=True):
     if print_text:
         print '\rChecking the config file...'
 
+    config_content = toml.loads(conf_file)
+    exclude = config_content.get("ignore", {})
+
+    # Check for invalid listed ignored directories
+    exclude_dirs = exclude.get("directories", [])
+    skip_dirs = ["./target", "./tests"]
```

Comment:
what about `./tests/wpt/sync` and `./tests/wpt/sync_css` which does not exist, but they are in [css](https://github.com/servo/servo/blob/master/tests/wpt/config_css.ini#L9) and [wpt](https://github.com/servo/servo/blob/master/tests/wpt/config.ini#L9) config files, or is that obsolete?